### PR TITLE
Add `has_bulk_rna_seq` to Sample model.

### PR DESCRIPTION
**Pull request by arkid15r**
#228 

New feature (non-breaking change which adds functionality)

Add the new field and populate it during data loading based on sample_id presence in the bulk metadata file.
Set project's `has_bulk_rna_seq` setting based on project's metadata file value.

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: api/scpca_portal/models/sample.py [hunk 1]
```diff
@@ -60,11 +61,12 @@ def create_from_dict(cls, data, project):
         sample = cls(
             additional_metadata=additional_metadata,
             age_at_diagnosis=data["age"],
-            cell_count=data["cell_count"],
+            cell_count=data.get("cell_count", 0),
```
Comment (davidsmejia) [inline api/scpca_portal/models/sample.py:64]: I think we need to confirm that this should be either 0 or null where we would display 'N/A' on the client.

Comment (arkid15r) [reply api/scpca_portal/models/sample.py:64]: We certainly do.
Would you prefer to address it in this PR or a separate one?

Comment (davidsmejia) [reply api/scpca_portal/models/sample.py:64]: Ill make an issue for this

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**